HACKING.md

```diff
@@ -2,36 +2,45 @@
 
 The partners.ubuntu.com codebase is a [Django](https://www.djangoproject.com/) app, which also includes some [NPM](https://www.npmjs.com/)-based tools and tasks for building static files like CSS.
 
-## Running the site with Docker
+## Running the site with Dotrun
 
-The recommended way to run the site is with the existing `./run` script, which uses Docker behind the scenes.
+The recommended way to run the site is by using the dotrun snap.
 
 First [install Docker](https://docs.docker.com/engine/installation/) and add your user to the `docker` group.
+Secondly, install dotrun
+```
+snap install dotrun
+```
+
+You can then run the site by simply typing within the repository:
+``` bash
+dotrun
+```
```

Comment:
Could you maybe add a line here explaining that once you run the site you have to go to `http://0.0.0.0:8003/admin`, because `http://0.0.0.0:8003/` itself will just redirect to canonical.com/partners ?

Reply:
Great suggestion. Done!